nubis/bin/consul_inputs.sh

```diff
@@ -40,19 +40,31 @@ get-route53-nameservers () {
 
 update-consul () {
     if [ ! -f ${IO_FILE:-0} ]; then
-        Echo "ERROR: Must pass a json file to update-consul"
+        echo "ERROR: Must pass a json file to update-consul"
         exit 1
     fi
-    wgServer=$(jq --monochrome-output --raw-output '.[0][] | if .OutputKey == "wgServer" then .OutputValue else empty end | @text' $IO_FILE)
-    wgDBserver=$(jq --monochrome-output --raw-output '.[0][] | if .OutputKey == "wgDBserver" then .OutputValue else empty end | @text' $IO_FILE)
-    wgDBname=$(jq --monochrome-output --raw-output '.[0][] | if .OutputKey == "wgDBname" then .OutputValue else empty end | @text' $IO_FILE)
-    wgDBuser=$(jq --monochrome-output --raw-output '.[0][] | if .OutputKey == "wgDBuser" then .OutputValue else empty end | @text' $IO_FILE)
 
-    CONSUL="http://$CONSUL_ENDPOINT:8500/v1/kv/$PROJECT_NAME/$NUBIS_ENVIRONMENT/config"
-    curl -s -X PUT -d $wgServer $CONSUL/wgServer
-    curl -s -X PUT -d $wgDBserver $CONSUL/wgDBserver
-    curl -s -X PUT -d $wgDBname $CONSUL/wgDBname
-    curl -s -X PUT -d $wgDBuser $CONSUL/wgDBuser
+    CONSUL="http://ui.$CONSUL_ENDPOINT/v1/kv/$PROJECT_NAME/$NUBIS_ENVIRONMENT"
```

Comment:
ATM, the reason for this is that there are 2 DNS names for a given Consul cluster.

One for Consul gossip discovery itself, and another one, fronted by an ELB for the web-ui/api.

They can't be the same because ELBs don't do UDP and all. Wonder how this could be more proprely abstracted away.